azure_functions_worker/dispatcher.py

```diff
@@ -501,6 +519,15 @@ def tp_max_workers_validator(value: str) -> bool:
             default_value=f'{PYTHON_THREADPOOL_THREAD_COUNT_DEFAULT}',
             validator=tp_max_workers_validator))
 
+    def _create_sync_call_tp(
+            self, max_worker: int) -> concurrent.futures.Executor:
+        """Update the self._sync_call_tp executor with the proper max_worker.
```

Comment:
Can you update the comments as this is now more specifically used to create an executor